spec.html

```diff
@@ -1690,14 +1690,39 @@ <h1>SetValueInBuffer ( _arrayBuffer_, _byteIndex_, _type_, _value_, _isTypedArra
           1. If IsSharedArrayBuffer(_arrayBuffer_) is *true*, then
             1. Let _execution_ be the [[CandidateExecution]] field of the surrounding agent's Agent Record.
             1. Let _eventList_ be the [[EventList]] field of the element in _execution_.[[EventLists]] whose [[AgentSignifier]] is AgentSignifier().
-            1. If _isTypedArray_ is *true* and _type_ is `"Int8"`, `"Uint8"`, `"Int16"`, `"Uint16"`, `"Int32"`, or `"Uint32"`, let _noTear_ be *true*; otherwise let _noTear_ be *false*.
+            1. If _isTypedArray_ is *true* and _type_ is `"Int8"`, `"Uint8"`, `"Int16"`, `"Uint16"`, `"Int32"`, or `"Uint32"`, <ins>or if _type_ is `"BigInt64"` or `"BigUint64"` and _order_ is `"SeqCst"`</ins>, let _noTear_ be *true*; otherwise let _noTear_ be *false*.
```

Comment:
How about "_order_ is not `"Init"` or `"Unordered"`, to make it clear that there's nothing magic about the memory order SeqCst in this regard?  (Ie if we were to add some acquire-release order it would be covered by this clause without changes being required, which is how it should be.)

Reply:
Is there any chance that we'd add some other value for *order* which is also weak?

Reply:
I think that if we manage to resurrect SAB+Atomics and people start using it, either in JS by itself or to interact in serious ways with wasm, then an acquire-release order would probably appear, and would have a different value for the order parameter.

Reply:
OK, switched to the wording you proposed.